schema/virtual-memory-extension.asn

```diff
@@ -0,0 +1,19 @@
+Virtual-Memory-Extension
+DEFINITIONS
+   AUTOMATIC TAGS ::=
+BEGIN
+   EXPORTS VirtMem;
+
+   VirtMem ::= SEQUENCE {
+      -- 1 bit to indicate Svnapot standard extension is supported on Sv39, Sv48
+      -- and Sv57 page table entry.
+      svnapot BOOLEAN,
+
+      -- 1 bit to indicate Svpbmt standard extension is supported on Sv39, Sv48
+      -- and Sv57 page table entry.
+      svpbmt  BOOLEAN,
+
+      -- 1 bit to indicate Svinval Standard Extension is supported.
+      svinval BOOLEAN
+   }
```

Comment:
We should end this SEQUENCE with `, ...` unless we're absolutely sure there will never be any more information from the virtual memory group.

Reply:
sure, done in the second commit